batcher_test.go

```diff
@@ -36,21 +36,43 @@ func TestBatch_Timeout(t *testing.T) {
 		t.Fatal("failed to create batcher for timeout test")
 	}
 
-	in, out := batcher.Start()
+	batcher.Start()
 
 	var p Point
 	go func() {
 		for i := 0; i < batchSize; i++ {
-			in <- p
+			batcher.In() <- p
 		}
 	}()
-	batch := <-out
+	batch := <-batcher.Out()
 	if len(batch) != batchSize {
 		t.Errorf("received batch has incorrect length exp %d, got %d", batchSize, len(batch))
 	}
 	checkPointBatcherStats(t, batcher, -1, batchSize, 0, 1)
 }
 
+// TestBatch_Flush ensures that a batcher generates a batch when flushed
+func TestBatch_Flush(t *testing.T) {
+	batchSize := 2
+	batcher := NewPointBatcher(batchSize+1, time.Hour)
```

Comment:
Should it be batchSize=1?  Not sure I understand the +1 as it makes the batchsize 3 and then you test for 1 but initially set your var to 2.  Cut and paste?